prefect_aws/ecs.py

```diff
@@ -212,6 +212,15 @@ class ECSTask(Infrastructure):
             "unless `stream_output` is set. "
         ),
     )
+    cloudwatch_logs_options: Optional[Dict[str, str]] = Field(
+        default=None,
```

Comment:
Would default_factory work better here?

Reply:
I'm not sure what our standard is, generally I use a default factory but I think it basically depends how it changes rending in the UI here. I do not expect this to be used often so I tried to emphasize that it is nullable.